### ISSUE TITLE
Unable to resolve clikt 3.0.0

### ISSUE THREAD

**Issue by carltonwhitehead** [ajalt/clikt#235]
I followed the [steps to migrate to 3.0.0](https://ajalt.github.io/clikt/migration/#upgrading-to-30), but Maven can't resolve the artifact.

The dependency declared in my pom is:
```xml
<dependency>
    <groupId>com.github.ajalt.clikt</groupId>
    <artifactId>clikt</artifactId>
    <version>3.0.0</version>
</dependency>
```
There's an error building the module in my project with the above dependency:
```no-highlight
Could not find artifact com.github.ajalt.clikt:clikt:jar:3.0.0
```
I followed the URL just to sanity check Maven, and the URL given by the log definitely doesn't exist.
```no-highlight
https://repo.maven.apache.org/maven2/com/github/ajalt/clikt/clikt/3.0.0/clikt-3.0.0.jar
```
Should there be a jar published with the project?

**Comment by ajalt** [comment]
Clikt 3.0 uses the new gradle metadata available in Kotlin 1.4 to resolve the platform-specific artifacts. 

If you're using Maven, you'll need to use the [clikt-jvm](https://search.maven.org/search?q=a:clikt-jvm) artifact. I'll update the readme to mention that.

**Comment by carltonwhitehead** [comment]
Thanks! Problem solved.
